HardwareDeployment.md

```diff
@@ -0,0 +1,116 @@
+ 
+1. Gigabyte Brix GB-BXBT-1900
+![alt text](https://i1.wp.com/lightninginabox.co/wp-content/uploads/2018/08/20150603180458_m.png?resize=300%2C254&ssl=1 "Gigabyte Brix GB-BXBT-1900")
+2. Crucial 8GB DDR3L-1600 SODIMM
+![alt text](https://i0.wp.com/lightninginabox.co/wp-content/uploads/2018/08/204-pinsodimmddr3.png?resize=300%2C133&ssl=1 "Crucial 8GB DDR3L-1600 SODIMM")
+3. WD Blue 1TB Mobile Hard Disk Drive
+![alt text](https://i1.wp.com/lightninginabox.co/wp-content/uploads/2018/08/81qCyd2gDL._SL1500_.jpg?resize=300%2C300&ssl=1 "WD Blue 1TB Mobile Hard Disk Drive")
+
+
+Other requirements are as follows:
+
+1. High speed internet connection.
+2. Static IP
+3. Domain Name
+4. Ability to open ports (80, 443, 9735) on your router.
+5. Tiny Screwdriver
+6. USB Thumb Drive
+7. USB Keyboard, Mouse and Monitor (for initial install only). Can be headless when completed.
+8. Four Port USB Hub  
+
+Assuming you purchased the hardware mentioned above, here are the build instructions.
+
+**Step 1** - Configure your domain name. 
+It can take several hours for DNS changes to propagate so you should do this step first.  Login to your domain registrar and point an A record from your domain to the external IP address of your internet connection.  I suggest that you use a subdomain (ie. btcpay.yourdomain.com).  If you already have your own Ubuntu Server you can skip to Step 6. 
+
+**Step 2** - Assemble your BTCPayBox (BTCPB)
+  - Remove back cover with screwdriver.
+  - Insert RAM
+  - Install hard drive using included cage. 
+
+**Step 3** - Download [Ubuntu 18.04 Desktop Image](http://releases.ubuntu.com/18.04/ubuntu-18.04.1-desktop-amd64.iso)
+
+**Step 4** - Download and install [Etcher](https://etcher.io/).  Etcher is software that is used to flash OS images to SD cards and USB Drives. 
+In this case we will be using Etcher to flash our USB Thumb Drive with the Ubuntu OS. 
+
+**Step 5** - Connect your USB keyboard, mouse, monitor and thumb drive.  Press the power button to boot your BTCPB using the thumb drive.  The Ubuntu installation process is pretty simple and easy to follow.  Here's a tutorial from the Ubuntu website.  [Install Ubuntu Desktop](https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-desktop#0)
+
+**Step 6** - Give your BTCPB a static IP address on your local network. There are a few different ways to do this and you will find a ton of articles online. Here's a pretty simple one to follow [How to configure a static IP address on Ubuntu 18.04](https://linuxconfig.org/how-to-configure-static-ip-address-on-ubuntu-18-04-bionic-beaver-linux).  To avoid conflicts with others devices on your network you should also set a "reservation" on your router so BTCPB's IP address doesn't get given out to another device on your network. 
```

Comment:
> To avoid conflicts with others devices...

Better as either:
To avoid conflicts with other devices
To avoid conflicts with others' devices